upi/metal/config.tf

```diff
@@ -176,6 +176,33 @@ EOF
 
 }
 
+variable "packet_plan" {
```

Comment:
Let's put a default here what we were using previously, "c1.small.x86". That way if there are current users of the UPI terraform, they can continue to use their flow without any changes.

Reply:
Fixed.